massa-node/Cargo.toml

```diff
@@ -22,7 +22,7 @@ sandbox = [
     "massa_models/sandbox",
     "massa_metrics/sandbox",
 ]
-testing = ["massa_metrics/testing", "massa_grpc/testing"]
+test-exports = ["massa_metrics/test-exports", "massa_grpc/test-exports"]
```

Comment:
Is there cases where we want to use this feature in the node?

Reply:
Fixed.